internal/labels.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/kiegroup/rhpam-kogito-operator/version"
+)
+
+var meteringLabels = map[string]string{
+	"com.company":   "Red_Hat",
+	"rht.prod_name": "Red_Hat_Process_Automation",
+	"rht.prod_ver":  trimMinorVersion(version.Version),
+	"rht.comp":      "PAM",
+	"rht.comp_ver":  version.PAMVersion,
+	"rht.subcomp":   "rhpam-kogito-runtime",
+	"rht.subcomp_t": "application",
+}
+
+// SetMeteringLabels for product operator
+func SetMeteringLabels(deploymentLabels map[string]string) map[string]string {
+	if deploymentLabels == nil {
+		deploymentLabels = map[string]string{}
+	}
+	for labelKey, labelValue := range meteringLabels {
+		deploymentLabels[labelKey] = labelValue
+	}
+	return deploymentLabels
+}
+
+func trimMinorVersion(version string) string {
```

Comment:
reuse below fcuntion in place of `trimMinorVersion`.
https://github.com/kiegroup/kogito-operator/blob/c03c9fd08199735d802a579eb4ca8cfc3c186db8/core/infrastructure/image.go#L166